files/build_templates/docker_image_ctl.j2

```diff
@@ -10,6 +10,7 @@ start() {
         docker start -a {{docker_container_name}}
     else
         docker run {{docker_image_run_opt}} \
+            -v /var/run/redis:/var/run/redis:rw \
```

Comment:
Shall we keep this in $(DOCKER_*)_RUN_OPT?
Is it possible that there might be some container that won't want to mount the database?